tests/x11/chromium.pm

```diff
@@ -27,27 +23,19 @@ sub run {
     ensure_installed 'chromium';
 
     # avoid async keyring popups
-    x11_start_program('chromium --password-store=basic', target_match => 'chromium-main-window', match_timeout => 50);
-
-    wait_screen_change { send_key 'esc' };    # get rid of popup (or abort loading)
-    send_key 'ctrl-l';    # select text in address bar
+    # allow key input before rendering is done, see poo#109737 for details
+    x11_start_program('chromium --password-store=basic --allow-pre-commit-input', target_match => 'chromium-main-window');
```

Comment:
why delete the `match_timeout`? I don't recall what the default match timeout is but I assume it was there for a purpose

Reply:
It's 90s